projects/plugins/jetpack/_inc/client/search/dashboard/module-control.jsx

```diff
@@ -80,6 +83,33 @@ function Search( props ) {
 	const togglingInstantSearch = !! props.isSavingAnyOption( 'instant_search_enabled' );
 	const isSavingEitherOption = togglingModule || togglingInstantSearch;
 
+	const isInstantSearchCFAButtonDisabled =
+		isSavingEitherOption ||
+		! isModuleEnabled ||
+		! isInstantSearchEnabled ||
+		! hasActiveSearchPurchase;
+
+	const renderInstantSearchCFAButtons = () => {
+		return (
+			<div className="jp-form-search-cfa-buttons">
+				<Button
+					className="jp-form-search-cfa-button customize-search-button"
+					href={ ! isInstantSearchCFAButtonDisabled && SEARCH_CUSTOMIZE_URL }
+					disabled={ isInstantSearchCFAButtonDisabled }
+				>
+					{ __( 'Customize search results', 'jetpack' ) }
+				</Button>
+				<Button
+					className="jp-form-search-cfa-button widgets-editor-button"
+					href={ ! isInstantSearchCFAButtonDisabled && WIDGETS_EDITOR_URL }
+					disabled={ isInstantSearchCFAButtonDisabled }
```

Comment:
I think this button should remain enabled as long as the module is enabled since Classic Search relies purely on the Widget editor to do its job.

What do you think?

Reply:
Yep, that's actually a good idea for the user's convenience. changed in 033de51e5.